pkg/api/add_events/add_events.go

```diff
@@ -26,13 +26,22 @@ import (
 	"github.com/scalyr/dataset-go/pkg/api/response"
 )
 
+const (
+	AttrBundleKye      = "bundle_key"
```

Comment:
Small type - Kye - Key

Reply:
Wow! Great catch. :)

It wasn't underscored, because it's valid word. :D 

https://www.merriam-webster.com/dictionary/kye - it's plural form and dialectical version of https://www.merriam-webster.com/dictionary/kine - which is archaic plural for cow - https://www.merriam-webster.com/dictionary/cow

Reply:
🐄 finally a valid use for this emoji :D